src/main/kotlin/no/nav/klage/oppgave/service/EndringService.kt

```diff
@@ -0,0 +1,48 @@
+package no.nav.klage.oppgave.service
+
+import no.nav.klage.oppgave.domain.klage.BehandlingSkygge
+import no.nav.klage.oppgave.domain.klage.Endring
+import no.nav.klage.oppgave.domain.klage.Endringstype
+import no.nav.klage.oppgave.domain.oppgavekopi.OppgaveKopiVersjon
+import no.nav.klage.oppgave.repositories.EndringRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+@Transactional
+class EndringService(
+    private val tokenService: TokenService,
+    private val klagebehandlingService: KlagebehandlingService,
+    private val endringRepository: EndringRepository,
+    private val oppgaveDiffService: OppgaveDiffService
+) {
+
+    fun getEndringerNotReadBySaksbehandler(saksbehandler: String) =
+        endringRepository.findBySaksbehandlerAndDatoLestIsNull(saksbehandler)
+
+    fun checkForEndring(oppgavekopier: List<OppgaveKopiVersjon>) {
+        if (oppgavekopier.size <= 1) {
+            return
+        }
+        val endring = oppgaveDiffService.diff(oppgavekopier[0], oppgavekopier[1])
+        if(endring != null) {
+            val klagebehandling = klagebehandlingService.getKlagebehandlingForOppgaveId(oppgavekopier[0].id).first()
```

Comment:
Dette er litt hacky foreløpig, for jeg er usikker: Kan vi oppleve at flere klagebehandlinger er knyttet til samme oppgave?